### PR TITLE
1st commit by sherif

### PR DESCRIPTION
**Pull request by shreo1992-beep**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<html> <body> hello <div> the </div> </body> </html>
```
Comment (khaledkz) [inline index.html]: Please Format your code!

Comment (shreo1992-beep) [reply index.html]: ok

Comment (shreo1992-beep) [reply index.html]: @khaledkz
we made the modification kindly check

**Review by khaledkz — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (mahmoud721) [reply index.html]: ok

**Pull request merged**